src/routes/wallet/wallet.routes.js

```diff
@@ -167,20 +170,10 @@ walletRouter.post(
 
 walletRouter.post(
   '/decode',
-  checkSchema({
-    txHex: {
-      in: ['body'],
-      errorMessage: 'Invalid txHex',
-      isString: true,
-      custom: {
-        options: (value, { req, location, path }) => {
-          // Test if txHex is actually hex
-          if (!(/^[0-9a-fA-F]+$/.test(value))) return false;
-          return true;
-        }
-      },
-    },
-  }),
+  oneOf([
+    checkSchema(txHexSchema),
+    checkSchema(partialTxSchema),
+  ], 'Required at least one of txHex or partial-tx'),
```

Comment:
Isn't it `partial_tx`? And should we change to `partialTx` so we keep the same format we've been using?

Reply:
Changed to `partial_tx`.

I kept this argument as snake case because we have other API arguments as snake case (e.g. `change_address`, `max_utxos`, `create_mint`) while `txHex` seems to be an exception.
But I have no problem changing this to `partialTx` since we already use this as the variable name.
What do you think?

Reply:
We can keep `partial_tx`, no problem